back/src/Model/GameRoom.ts

```diff
@@ -95,10 +96,20 @@ export class GameRoom {
         return Array.from(this.groups.values());
     }
 
+    public getGroupIncludingUser(user: User): Group | undefined {
+        const foundGroups = this.getGroups().filter((grp) => grp.includes(user));
+        return foundGroups[0];
+    }
```

Comment:
> But we can deal with improving that later. Maybe add another button at the bottom right?

Hmm for now that would actually be a better solution, I agree. But:

> At some point, we will refactor the UX so that a menu appear when you press [space] next to a WOKA. This way, we could have many actions available (like "follow", "view details", "send file", "send private message", etc...) But for now, I'm fine with starting pressing the interact button ([space]) to start following.

That's why I built it like this.

I thought about changing the the Svelte component "Interact menu" to a somewhat more specific  "Follow menu" or something like that and including it in a new generic "Interact" Svelte component. Instead I could just bind it to a new button that is always shown.

Reply:
Whoa, didn't see that. Thanks!